pysb/simulator/base.py

```diff
@@ -914,6 +914,112 @@ def observables(self):
             raise ValueError('Model has no observables')
         return self._squeeze_output(self._yobs)
 
+    def observable(self, pattern):
+        """
+        Calculate a pattern's trajectories without adding to model
+
+        Normally, observables are added in advance to a model. If a new
+        observable is required, previously it was necessary to re-run the
+        simulation. This method can calculate observables on demand using
+        any supplied MonomerPattern or ComplexPattern against the simulation
+        result, without re-running the simulation.
+
+        Note that the monomers within the supplied pattern are reconciled
+        with the SimulationResult's internal copy of the model by name. This
+        method only works on simulations which calculate species
+        trajectories (i.e. it will not work on network-free simulations).
+
+        Raises a ValueError if the pattern does not match at least one species.
+
+        Parameters
+        ----------
+        pattern: pysb.MonomerPattern or pysb.ComplexPattern
+            An observable pattern to match
+
+        Returns
+        -------
+        pandas.Series
+            Series containing the simulation trajectories for the specified
+            observable
+
+        Examples
+        --------
+
+        >>> from pysb.examples import earm_1_0
```

Comment:
I feel the example section should just succinctly demonstrate the extraction of a novel observable not defined in the model. Recapitulating an existing observable obscures the whole point of this cool feature!